changelog.md

```diff
@@ -1,3 +1,11 @@
+## Changes from 1.5.12 to 1.5.14
+
+## Introduce global throttling to Marathon health checks
+Marathon health checks is a deprecated feature and customers are strongly recommended to switch to Mesos health checks for scalability reasons. However, we've seen a number of issues when excessive number of Marathon health checks (HTTP and TCP) would overload parts of Marathon. Hence we introduced a new parameter `--max_concurrent_marathon_health_checks` that defines maximum number (256 by default) of *Marathon* health checks (HTTP/S and TCP) that can be executed concurrently in the given moment. Note that setting a big value here will overload Marathon leading to internal timeouts and unstable behavior.  
```

Comment:
Setting the value alone will not lead to timeouts, as this sentence seems to imply

Reply:
:clap: change log